src/config/xorg/session-management.md

```diff
@@ -0,0 +1,51 @@
+# Session and Seat Management
+
+Session and seat management is not necessary for every setup, it is used to
+provide device access on the fly for the currently active user session.
+
+For desktop environments like Gnome [elogind](#elogind) is necessary.
+
+## ConsoleKit2
+
+Install ConsoleKit2 and activate its service and make sure the `dbus` and the
+`cgmanager` services are activated too.
+
+```
+# xbps-install -S ConsoleKit2
+# ln -s /etc/sv/dbus /var/service/
+# ln -s /etc/sv/cgmanager /var/service/
+# ln -s /etc/sv/consolekit /var/service/
+```
+
+If you don't use a display manager or your display manager doesn't start a
```

Comment:
```suggestion
If you don't use a display manager, or your display manager doesn't start a
ConsoleKit2 session on its own, you need to start a ConsoleKit2 session from your
```
Add commas around condition.

Reply:
GitHub apparently won't let me suggest multi-line changes easily. This applies to both lines 20 and 21 of the diff.